applications/asset_tracker_v2/src/ext_sensors/Kconfig

```diff
@@ -9,6 +9,9 @@ menuconfig EXTERNAL_SENSORS
 
 if EXTERNAL_SENSORS
 
+config EXTERNAL_SENSORS_SET_ACCELEROMETER_TRIGGER_ON_INIT
+	bool "Set accelerometer trigger on initialization"
```

Comment:
Please add a description that explains what `true` and `false` mean for the application.
It should explain what should happen if this is set to `true` and what should happen if this is set to `false`, and what happens if is not defined (it looks like this could use a default value, so it has only two states, and not three).

Reply:
No problem, added a more descriptive "help" section for the Kconfig. Boolean Kconfigs are disabled by default without explicitly setting the default value.

Reply:
> Boolean Kconfigs are disabled by default without explicitly setting the default value.

Ah, I wasn't aware of that convention.